### PR TITLE
feat(HMS-2496): add no permission empty state

### PR DESCRIPTION
**Pull request by amirfefer**
Depends on https://github.com/RedHatInsights/image-builder-frontend/pull/1343

Screen Shot 2023-09-04 at 17 54 05

### CODE REVIEW

File: src/Components/ProvisioningWizard/index.js [hunk 1]
```diff
@@ -58,11 +59,14 @@ const ProvisioningWizard = ({ onClose, image, ...props }) => {
     setLaunchSuccess,
   });
 
-  const onNext = ({ id, name }, { prevId, prevName }) => {
-    console.debug(`current id: ${id}, current name: ${name}, previous id: ${prevId}, previous name: ${prevName}`);
```
Comment (amirfefer) [inline src/Components/ProvisioningWizard/index.js:62]: some old debug to clean...

**Comment by lzap** [comment]
So permission problem is resolved, I understand from Slack. Great. Linter issue.

**Comment by ezr-ondrej** [comment]
LGTM, but I guess we'll need to update the IQE tests, right @mayurilahane, @akhil-jha ? :)

**Comment by ezr-ondrej** [comment]
/retest

Last run https://ci.int.devshift.net/job/RHEnVision-provisioning-frontend-pr-check/621/

**Comment by mayurilahane** [comment]
> LGTM, but I guess we'll need to update the IQE tests, right @mayurilahane, @akhil-jha ? :)

yeah think we need to do some modal definition updates for this UI.
_if its new pop-up modal_

**Comment by ezr-ondrej** [comment]
/retest

Huh? Not a new modal, more like a new step in current modal for users without perms

**Comment by amirfefer** [comment]
It's not a new modal (the modal is created on the IB side), though the new addition in IB  https://github.com/RedHatInsights/image-builder-frontend/pull/1343 adds a mandatory  `hasAccess` prop. We need to ensure that in the ephemeral test environment, we have the latest IB and configured RBAC service with provisioning-write, in order to render the wizard.

**Comment by mayurilahane** [comment]
As we do not have any RBAC UI test cases right now it will be safe to merge this.

**Review by ezr-ondrej — APPROVED** [review]
We do not test RBAC in UI yet  (luckily, as we should test this screen).
Thanks @amirfefer nice work! :+1:

**Comment by ezr-ondrej** [comment]
And thanks @mayurilahane for confirmation! :tada:

**Pull request merged**